fastlane/Fastfile

```diff
@@ -28,9 +28,18 @@ platform :ios do
 
   desc "Increment build number"
   lane :bump do |options|
-    version_number = options[:version]
-    increment_version_number(version_number: version_number)
-    version_bump_podspec(version_number: version_number)
+    new_version_number = options[:version]
+    previous_version_number = get_version_number
+    increment_version_number(version_number: new_version_number)
+    version_bump_podspec(version_number: new_version_number)
+    increment_build_number_in_rc_purchases(previous_version_number, new_version_number)
```

Comment:
this is the interesting line here, where we update the version number in RCPurchases.m